antarest/tools/admin_lib.py

```diff
@@ -61,3 +72,34 @@ def reindex_table(config: Path) -> None:
         connection.execute(text("REINDEX INDEX study_pkey"))
         connection.execute(text("VACUUM ANALYSE rawstudy"))
         connection.execute(text("REINDEX INDEX rawstudy_pkey"))
+
+
+def _do_fix_interrupted_tasks_status(engine: Engine) -> None:
+    updated_values = {
+        TaskJob.status: TaskStatus.FAILED.value,
+        TaskJob.result_status: False,
+        TaskJob.result_msg: "Task was interrupted due to server restart",
+        TaskJob.completion_date: datetime.now(timezone.utc).replace(tzinfo=None),
+    }
+    orphan_status = [TaskStatus.RUNNING.value, TaskStatus.PENDING.value]
+    logger.info("Setting status of interrupted tasks to FAILED.")
+    t0 = time.time()
+    with engine.connect() as connection:
+        stmt = update(TaskJob).where(TaskJob.status.in_(orphan_status)).values(updated_values)
+        connection.execute(stmt)
+        connection.commit()
+    logger.info(f"Tasks status correctly updated in {time.time() - t0}s.")
+
+
+def fix_interrupted_tasks_status(config_file: Path) -> None:
```

Comment:
I suggest error handling in this method in case of timeout, connexion error ...

Reply:
I think the global behaviour is OK, so I am not sure we should make things more complex:
if there is an exception here, the command line tool will return an invalid exit code, which will make the startup script fail, and will cause either a retry (the service will try to restart), or a human analysis if the problem persists.